spec/support/helpers/javascript_helpers.rb

```diff
@@ -17,4 +17,10 @@ def wait_for_device_info(form_name)
     end
   end
 
+  def simulate_submission_pdf_ready
```

Comment:
why does NJ need it’s own javascript helper?

Reply:
I think this is used by every state